Ubuntu1804-readme.md

```diff
@@ -1 +1,318 @@
+# Ubuntu 18.04.5 LTS
+- Image Version: 20200820.1
 
+## Installed Software
+### Basic packages
+- dnsutils
+- iproute2
+- iputils-ping
+- libunwind8
+- locales
+- openssh-client
+- tzdata
+- upx
+- zstd
+- libxkbfile-dev
+- pkg-config
+- libsecret-1-dev
+- libxss1
+- libgconf-2-4
+- dbus
+- xvfb
+- libgbm-dev
+- libgtk-3-0
+- tk
+- fakeroot
+- dpkg
+- rpm
+- xz-utils
+- xorriso
+- zsync
+- gnupg2
+- lib32z1
+- texinfo
+- libsqlite3-dev
+- libc++-dev
+- libc++abi-dev
+- curl
+- file
+- ftp
+- jq
+- netcat
+- ssh
+- parallel
+- rsync
+- shellcheck
+- sudo
+- telnet
+- time
+- unzip
+- zip
+- wget
+- m4
+- bison
+- flex
+- patchelf
+- bzip2
+- sqlite3
+- brotli
+- yamllint
+- libcurl3
+
+### Language and Runtime
+- GNU C++ 7.5.0
+- GNU C++ 8.4.0
+- GNU C++ 9.3.0
+- GNU Fortran 8.4.0
+- GNU Fortran 9.3.0
+- Clang 6.0 (6.0.0)
+- Clang 8 (8.0.0)
+- Clang 9 (9.0.0)
+- Erlang 11.0.3
+- Mono 6.10.0.104
+- Node.js 12.18.3
+- Python 2.7.17
+- Python3 3.6.9
+- Powershell 7.0.3
+- Ruby 2.5.1p57
+- Swift 5.2.5
+- Julia 1.5.0
+
+### Package Management
+- Homebrew 2.4.13
+- Gem 3.1.4
+- Miniconda 4.8.3
+- Helm 3.3.0
+- NPM 6.14.6
+- Yarn 1.22.4
+- pip 9.0.1 (python 2.7)
+- pip3 9.0.1 (python 3.6)
+- Vcpkg 2020.06.15
+
+### Project Management
+- Ant 1.10.5
+- Gradle 6.6.1
+- Maven 3.6.3
+- sbt 1.3.13
+
+### Tools
+- 7-Zip 16.02
+- Ansible 2.9.12
+- AzCopy7 (available by azcopy alias) 7.3.0
+- AzCopy10 (available by azcopy10 alias) 10.6.0
+- Bazel 3.4.1
+- Bazelisk 1.6.1
+- Buildah 1.15.1
+- CMake 3.17.0
+- Docker Compose 1.26.2
+- Docker-Moby 19.03.12
+- Docker-Buildx 0.4.2
+- Git 2.28.0
+- Git LFS 2.11.0
+- Git-ftp 1.3.1
+- Google Cloud SDK 307.0.0
+- Haveged 1.9.1
+- Heroku 7.42.10
+- HHVM (HipHop VM) 4.71.0
+- Podman 2.0.4
+- Skopeo 1.1.1
+- svn 1.9.7
+- Kustomize 3.8.1
+- Kind 0.8.1
+- Kubectl 1.18.8
+- Minikube 1.12.3
+- Mercurial 4.5.3
+- MongoDB 4.4.0
+- Leiningen 2.9.4
+- Newman (5.1.2)
+- nvm 0.35.3
+- Packer 1.6.1
+- PhantomJS 2.1.1
+- PostgreSQL 12.4
```

Comment:
Can we move DBs to a separate section like for Windows image? and add a bit more information about it if applicable

Reply:
Done